source/features/batch-open-conversations.tsx

```diff
@@ -63,12 +63,49 @@ async function init(): Promise<void | false> {
 				Open selected
 			</button>
 		);
+	} else {
+		const issuesToolbar = select('#js-issues-toolbar')!;
+		issuesToolbar.prepend(
+			<div className="mr-3 d-none d-md-block">
+				<input type="checkbox" data-check-all="" aria-label="Select all issues" autoComplete="off"/>
+			</div>
+		);
+		issuesToolbar.append(
+			<div className="table-list-triage flex-auto js-issues-toolbar-triage">
+				<span className="text-gray table-list-header-toggle">
+					<span data-check-all-count="">1</span> selected
+					{' '}
+					<button type="button" className="btn-link rgh-batch-open-issues pl-3">Open selected</button>
+				</span>
+			</div>
+		);
+
+		const regExpPrNumber = /^issue_(\d*)$/;
+
+		const prRows = select.all('.js-issue-row');
+		for (const prRow of prRows) {
+			const prNumber = regExpPrNumber.exec(prRow.id)?.[1] ?? '';
+			const prependAt = prRow.children[0]!;
+			prependAt.prepend(
+				<label className="flex-shrink-0 py-2 pl-3  d-none d-md-block">
+					<input
+						type="checkbox"
+						data-check-all-item=""
+						className="js-issues-list-check"
+						name="issues[]"
+						value={prNumber}
+						aria-labelledby={`issue_${prNumber}_link`}
+						autoComplete="off"
+					/>
+				</label>
+			);
+		}
 	}
 }
 
 void features.add({
 	id: __filebasename,
-	description: 'Adds a button to open multiple conversations at once in your repos.',
+	description: 'Adds a button to open multiple conversations at once.',
```

Comment:
I'm not a fan of _"Adds a button to..."_ descriptions, let's take this opportunity to improve it
```suggestion
	description: 'Lets you open multiple conversations at once via checkboxes.',
```